src/wp-includes/functions.wp-scripts.php

```diff
@@ -193,6 +195,9 @@ function wp_register_script( $handle, $src, $deps = array(), $ver = false, $args
 	if ( ! empty( $args['strategy'] ) ) {
 		$wp_scripts->add_data( $handle, 'strategy', $args['strategy'] );
 	}
+	if ( ! empty( $args['fetchpriority'] ) ) {
+		$wp_scripts->add_data( $handle, 'fetchpriority', $args['fetchpriority'] );
```

Comment:
The fetchpriority is checked against an allowlist for modules. I don't see that for scripts. Did I overlook that or should it be included?

Reply:
@sirreal In the case of classic scripts, the `fetchpriority` is instead being specified when the scripts are registered. The only script which gets `low` is `comment-reply`:

https://github.com/WordPress/wordpress-develop/blob/75fb879fd742318830982fe61afe010cf7afeb44/src/wp-includes/script-loader.php#L1050

There isn't any allowlist to default to `low` for non-module scripts since there is no generalizable pattern for being able to opt in like there is for script modules with the Interactivity API.

Reply:
More than default I'm wondering about _accepting_ arbitrary values. It's not likely harmful, but perhaps `_doing_it_wrong` could be shown and the value discarded if an unsupported `fetchpriority` is provided.

[I understand it's a well defined set of values](https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement/fetchPriority): `high | low | auto`.

For example:

```php
wp_enqueue_script( 'example', '/ex.js', array(), '…' );
wp_scripts()->add_data( 'example', 'fetchpriority', 'silly' );
```

produces

```html
<script src="/ex.js?ver=…" id="example-js" fetchpriority="silly"></script>
```

which is… silly 🙂